bin/migrate-mongo.js

```diff
@@ -17,11 +17,15 @@ function handleError(err) {
   process.exit(1);
 }
 
-function printStatusTable(statusItems) {
+async function printStatusTable(statusItems) {
   return migrateMongo.config.read().then(config => {
     const useFileHash = config.useFileHash === true;
     const table = new Table({ head: useFileHash ? ["Filename", "Hash", "Applied At"] : ["Filename", "Applied At"]});
-    statusItems.forEach(item => table.push(_.values(item)));
+    statusItems.forEach(item => {
```

Comment:
We don't wan to show the file contents in the status list, so we remove it from the returned status items